src/Raven.Server/Commercial/LicenseStatus.cs

```diff
@@ -86,14 +86,60 @@ public LicenseType Type
             }
         }
 
-        public int Version => GetValue<int?>("version") ?? -1;
+        public string Version
+        {
+            get
+            {
+                if (Attributes == null)
+                    return null;
+
+                if (Attributes.TryGetValue("version", out object versionValue) == false)
+                    return null;
+
+                switch (versionValue)
+                {
+                    case int versionInt:
+                        if (versionInt <= 10)
+                            return null;
+
+                        var versionDouble = versionInt / 10d;
+                        return versionDouble.ToString("F1");
+
+                    case string versionString:
+                        return versionString;
+
+                    default:
+                        return null;
+                }
+            }
+        }
+
+        public bool UpgradeRequired
+        {
+            get
+            {
+                if (Type != LicenseType.Community)
+                    return false;
+
+                if (Version == null)
+                    return false;
+
+                if (System.Version.TryParse(RavenVersionAttribute.Instance.Version, out var currentVersion) == false)
+                    return false;
+
+                var licenseVersion = new Version(Version);
+                return licenseVersion > currentVersion;
```

Comment:
That is super early, no?
Can we add "Minimum version to upgrade" or the like?

Reply:
I can add something like this, but the mechanism for generating the value will be completely the same as the mechanism implemented above for the `Version` property.
We specify the `version` in the license attribute, and in the case of the Community license, this is the very "Minimum version to upgrade".

Reply:
@ayende under Version we do have `5.4` or `6.0` so it will get parsed as 5.4.0 or 6.0.0
@ArieSLV shouldn't this be `licenseVersion >= currentVersion` ?

Reply:
> @ArieSLV shouldn't this be `licenseVersion >= currentVersion` ?
No, if we are already on the latest version, then an update is not required.

Reply:
Okay